SetupIncTraceLibs.bat

```diff
@@ -28,7 +28,15 @@ call spatial version
 call :MarkStartOfBlock "Unpack dependencies"
     powershell -Command "Expand-Archive -Path \"%CORE_SDK_DIR%\trace_lib\trace-win32.zip\"	-DestinationPath \"%BINARIES_DIR%\Win64\" -Force;"^
 						"Expand-Archive -Path \"%CORE_SDK_DIR%\trace_lib\trace-linux.zip\"	-DestinationPath \"%BINARIES_DIR%\Linux\" -Force;"
-	xcopy /s /i /q "%BINARIES_DIR%\Win64\improbable" "%WORKER_SDK_DIR%\improbable"
+	xcopy /s /i /q "%BINARIES_DIR%\Win64\improbable" "%WORKER_SDK_DIR%\improbable\legacy"
+
+	set LEGACY_FOLDER=%WORKER_SDK_DIR%\improbable\legacy\
+	set TRACE_HEADER="%LEGACY_FOLDER%trace.h"
+	set TEMP_HEADER="%LEGACY_FOLDER%temp.h"
+
+	cat %TRACE_HEADER% | sed -e "s|#include <improbable/c_trace.h>|#include <improbable/legacy/c_trace.h>|" > %TEMP_HEADER%
```

Comment:
`sed` can edit in place with `-i`, that should remove the need for the temp file.